webview-ui/src/components/chat/ChatTextArea.tsx

```diff
@@ -706,8 +706,8 @@ const ChatTextArea = forwardRef<HTMLTextAreaElement, ChatTextAreaProps>(
 					return `openai-compat:${selectedModelId}`
 				case "vscode-lm":
 					return `vscode-lm:${apiConfiguration.vsCodeLmModelSelector ? `${apiConfiguration.vsCodeLmModelSelector.vendor ?? ""}/${apiConfiguration.vsCodeLmModelSelector.family ?? ""}` : unknownModel}`
-				case "together":
-					return `${selectedProvider}:${apiConfiguration.togetherModelId}`
+				case "fireworks":
```

Comment:
Can we please not remove the together case?

Reply:
@mattapperson Can you work on this please?

Reply:
```suggestion
				case "together":
					return `${selectedProvider}:${apiConfiguration.togetherModelId}`
				case "fireworks":
```